client/coral-admin/src/components/Layout.css

```diff
@@ -1,4 +1,6 @@
 .layout {
   margin: 0 auto;
   background-color: #FAFAFA;
-}
+  height: inherit;
+  min-height: 100vh;
```

Comment:
As layout does not start at height: 0, this always means height > 100% and we'll always have a scrollbar.

Reply:
Give it sth like `300px`, this should be all you need to fix  https://www.pivotaltracker.com/n/projects/1863625/stories/156203156

Reply:
I don't think this is a solution. If someone adds another link this could potentially be a bug again.